jsons/1.11.2.json

```diff
@@ -558,6 +558,6 @@
     "minecraftArguments": "--username ${auth_player_name} --version ${version_name} --gameDir ${game_directory} --assetsDir ${assets_root} --assetIndex ${assets_index_name} --uuid ${auth_uuid} --accessToken ${auth_access_token} --userType ${user_type} --versionType ${version_type}",
     "minimumLauncherVersion": 18,
     "releaseTime": "2016-12-21T09:29:12+00:00",
-    "time": "2017-01-12T10:44:50+00:00",
+    "time": "2017-02-07T13:18:39+00:00",
```

Comment:
Every time you set up the project on 1.11.2 it updates this file, they probably commited it by accident.  
It should probably be git-ignored or updated, either way not a big deal as far as I know.